src/ucp/wireup/wireup.c

```diff
@@ -2039,7 +2230,8 @@ ucs_status_t ucp_wireup_connect_remote(ucp_ep_h ep, ucp_lane_index_t lane)
     ucp_wireup_ep_set_next_ep(ucp_ep_get_lane(ep, lane), uct_ep,
                               ucp_ep_get_rsc_index(ep, lane));
 
-    if (!(ep->flags & UCP_EP_FLAG_CONNECT_REQ_QUEUED)) {
+    if (!(ep->flags & UCP_EP_FLAG_CONNECT_REQ_QUEUED) &&
+        !(ep->flags & UCP_EP_FLAG_PRIORITY_UPDATE_QUEUED)) {
```

Comment:
```suggestion
    if (!(ep->flags & (UCP_EP_FLAG_CONNECT_REQ_QUEUED |
                       UCP_EP_FLAG_PRIORITY_UPDATE_QUEUED))) {
```
also I find these flags goes together in several cases, maybe define a mask?